e2e_tests/src/e2e_common/util.py

```diff
@@ -121,6 +122,26 @@ def xrun(cmd, *args, **kwargs):
         )
 
 
+def nxrun(func, **kwargs):
+    """
+    nxrun = NOT xrun
+    Like xrun but for executing python code and to
+        * call directly
+        * without a timeout
+        * with **kwargs only
+    """
+    fname = func.__name__
+    try:
+        result = func(**kwargs)
```

Comment:
I think yes its better to do it this way over using subprocess.

Reply:
for calling non-python - no such luck